src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,50 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "cinema_halls")
```

Comment:
Try to stick to the common conventions about table naming (it's usually in singular form).